test/malli/clj_kondo_test.cljc

```diff
@@ -78,7 +87,13 @@
                 :tuple-of-ints :nilable/seqable}}
          (clj-kondo/transform Schema)))
 
-  (let [expected-out
+  (let [it-fn-defs ['kikka
```

Comment:
I wanted to have a separate test for this specific issue, so that it could have it's own assertion w/ docstring. While doing so, this test broke because it's testing the entire ns, which seemed unnecessary. I would think as this test grew bigger, the chances for errors/mistakes to sneak through in a larger and larger assertion might grow, particularly with no docstrings explaining these. /shrug

Reply:
Thanks for thinking about this. We now have two methods for testing which adds complexity (extra filtering list of symbols to be tested with 1). I would choose just one way to test these:

1. Lean/Coherent: just add the new cases into to the existing assertion map. One could add comments above the expected map entry as extra info. Other malli test namespaces have this, e.g. swagger-tests so it's a common testing pattern. Adding a new case (here, a new `defn`) means the existing tests would break, but it's a good and automatic indication you have to add a new expectation too. Less code, but also less info/context on errors.
2. Normal: a helper that allows to test each defn separately in a own `deftest` / `testing` or `is` block. athen break down `clj-kondo-integration-test`.

I'm ok with both, but as a minimalist, would have done 1 myself.

Could you migrate the full test ns to use either one?

Reply:
If other test ns' are already following this pattern, I think consistency is worth going for, so I'll just update it to do (1).

Personally, though, I don't really like using comments for this kinda thing. My problems w/ it are:

- I have a tendency to just copy-paste 'actual' test output back into the test if it matches what I expect, and that ends up erasing the comments. (might just be a 'me' problem :) )
- Generally less UX friendly than having the str actually show up in the test failure output, extra legwork to cross-examine and figure things out. Relatively minor issue /shrug.